pkg/codegen/testing/test/testdata/dashed-import-schema/docs/tree/v1/nursery/_index.md

```diff
@@ -14,8 +14,149 @@ no_edit_this_page: true
 
 
 
+## Creating a Nursery Resource
+<div>
+<pulumi-chooser type="language" options="typescript,python,go,csharp,java,yaml"></pulumi-chooser>
+</div>
 
-## Create Nursery Resource {#create}
+<div>
+    <pulumi-choosable type="language" values="javascript,typescript">
+        <div class="highlight">
+            <pre class="chroma">
+                <code class="language-typescript" data-lang="typescript">
+import * as pulumi from "@pulumi/pulumi";
+import * as plant from "@pulumi/plant";
+
+const nursery = new plant.tree.v1.Nursery("nursery", {
+  sizes: {
+    "string": "small"|"medium"|"large"
+  },
+  varieties: ["Burgundy"|"Ruby"|"Tineke"],
+});
+
+                </code>
+            </pre>
+        </div>
+    </pulumi-choosable>
+</div>
+
+<div>
+    <pulumi-choosable type="language" values="python">
+        <div class="highlight">
+            <pre class="chroma">
+                <code class="language-python" data-lang="python">
+import pulumi
+import pulumi_plant as plant
+
+nursery = plant.tree.v1.Nursery("nursery",
+  sizes={
+    'string': "small"|"medium"|"large"
```

Comment:
I think this reads better as `"small" or "medium" or "large"`. 

Also, it works, returning `"small"`.